ci/integration/deploy-pipeline.yaml

```
@@ -51,6 +51,103 @@ spec:
 
     jobs:
 
+    - name: deploy-dk-integration
+      serial: true
+      plan:
+
+      - get: release
+        trigger: true
+
+      - get: daily
+        trigger: true
+
+      - task: render-manifests
+        config:
+          platform: linux
+          image_resource: *task_toolbox
+          inputs:
+          - name: release
+          outputs:
+          - name: manifests
+          params:
+            CLUSTER_NAME: ((cluster.name))
+            CLUSTER_DOMAIN: ((cluster.domain))
+            RELEASE_NAME: dk-integration
+            CONNECTOR_NODE_NATIONALITY_CODE: DK
+            RELEASE_NAMESPACE: ((namespace-deployer.namespace))
+            CLUSTER_PRIVATE_KEY: ((cluster.privateKey))
+            ERROR_PAGE_URL: https://www.integration.signin.service.gov.uk/proxy-node-error
+            CONNECTOR_ENTITY_ID: https://eidasconnector.test.eid.digst.dk/Metadata
+            CONNECTOR_METADATA_FQDN: eidasconnector.test.eid.digst.dk
```

```suggestion
            CONNECTOR_METADATA_FQDN: eidasconnector.test.eid.digst.dk
            HUB_FQDN: www.integration.signin.service.gov.uk
```
I think the `HUB_FQDN` was added to nl-integration recently, and we should add this too